### PR TITLE
updated icons to reflect new logo

### PR DESCRIPTION
**Pull request by theycallmemac**

### CODE REVIEW

**Comment by butlerx** [comment]
favicon has the wrong filetype it should be an icon not a png

**Comment by theycallmemac** [comment]
Isn't it there, favicon.ico?

**Comment by butlerx** [comment]
It's file type is ping not icon

File: data/schedule.yaml [hunk 1]
```diff
@@ -33,12 +34,12 @@ locations:
         by: Rory Ward (Google)
         time: '10:30'
         description: 'Scaling-up a System'
-      - name: >
```
Comment (butlerx) [inline data/schedule.yaml]: dont drop the >

File: data/schedule.yaml [hunk 2]
```diff
@@ -33,12 +34,12 @@ locations:
...
+      - name:
               HEAnet's New Network - 
               The Story So Far
         by: Brian Nisbet (HEAnet)
         time: '11:30'
-        description: >
```
Comment (butlerx) [inline data/schedule.yaml]: same as above

File: data/schedule.yaml [hunk 3]
```diff
@@ -73,17 +74,18 @@ locations:
...
+        description: 
                 Planning to attend this workshop?
                 Comeplete the pre-work here -
                 https://sistem.redbrick.dcu.ie/assets/pre_work.pdf
-      - name: SLOT FILLED
+      - name: Microsoft - Building a Modern SPA with Typescript and DotNet.
         by: TBA
         time: '11:30'
-      - name: Problem Solving with Code
+        description: Modern Web Apps are all the rage these days. Through this workshop you’ll get hands-on experience with some powerful technologies powering Microsoft’s SPA’s such as Typescript, ASP.NET Web API, React and Webpack. Combine these to develop a Url Minifier, and, get an insight into the skills needed to become a Full stack developer at Microsoft.
```
Comment (butlerx) [inline data/schedule.yaml]: use multi-line string rather the one line 

note: multiline string are the `>` ones

File: data/schedule.yaml [hunk 4]
```diff
@@ -73,17 +74,18 @@ locations:
...
                 Planning to attend this workshop?
                 Comeplete the pre-work here -
                 https://sistem.redbrick.dcu.ie/assets/pre_work.pdf
-      - name: SLOT FILLED
-        by: TBA
+      - name: Microsoft - Building a Modern SPA with Typescript and DotNet.
+        by: Siddarth Sharma
         time: '11:30'
-      - name: Problem Solving with Code
+        description: Modern Web Apps are all the rage these days. Through this workshop you’ll get hands-on experience with some powerful technologies powering Microsoft’s SPA’s such as Typescript, ASP.NET Web API, React and Webpack. You'll combine these to develop a Url Minifier, and, get an insight into the skills needed to become a Full stack developer at Microsoft.
```
Comment (butlerx) [inline data/schedule.yaml]: Use multiline strings rather than one

Comment (butlerx) [reply data/schedule.yaml]:
```yaml
      - name: Microsoft - Building a Modern SPA with Typescript and DotNet.
        by: Siddarth Sharma
        time: '11:30'
        description: >
          Modern Web Apps are all the rage these days. Through this workshop
          you’ll get hands-on experience with some powerful technologies
          powering Microsoft’s SPA’s such as Typescript, ASP.NET Web API,
          React and Webpack. You'll combine these to develop an Url Minifier,
          and, get an insight into the skills needed to become a Full stack
          developer at Microsoft.
      - name: Google - Problem Solving with Code
        by: Ohad Lutzky (Google)
        time: '14:15'
        description: >
          Google brings us a problem-solving workshop, showing you best practices
          for how to go about solving problems with code.
```

**Pull request merged**